format/index.md

```diff
@@ -258,6 +267,11 @@ pagination links, as described below.
 If a link object refers to resource objects included in the same compound
 document, it **MUST** include object linkage to those resource objects.
 
+> Note: If present, a *related resource URL* must be a valid URL, even if the
+relationship isn't currently associated with any target resources. If a
+relationship is empty, the server can indicate this in the link object by
+including an `id: null` or `id: []` member depending on relationship cardinality.
+
```

Comment:
Let's keep the first sentence, but not the second. The second is covered in the fetching relationships section (and btw, `"data": null` and `"data": []` are valid responses for empty relationships)